spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -478,7 +482,7 @@ public DeclaredQuery getQuery() {
 	 * </ul>
 	 */
```

Comment:
Please add the new `MERGE` to the JavaDoc of the enum.

`<li>{@code ParsedType.MERGE}: means the top level statement is {@link Merge}</li>`